src/components/Cart/BillingForm.tsx

```diff
@@ -66,42 +70,40 @@ export const BillingForm = () => {
     [cities],
   );
 
+  const BillingFormField = ({ name, label }: BillingFormFieldProps) => {
+    return (
+      <Controller
+        control={control}
+        name={name}
+        render={({ field, fieldState }) => (
+          <FormField
+            label={label}
+            placeholder={label}
+            {...field}
+            error={fieldState.error?.message}
+          />
+        )}
+      />
+    );
+  };
```

Comment:
Why do you need this wrapper here?

Reply:
@VYaniv , do you mean BillingFormField? It helps reduce repetitive code.

Reply:
Yes, but let`s discuss it